### PR TITLE
Is desktop

### PR DESCRIPTION
**Pull request by microArtur**
https://nanopay.atlassian.net/browse/NP-9454

### CODE REVIEW

File: src/foam/nanos/controller/ApplicationController.js [hunk 1]
```diff
@@ -224,6 +225,13 @@ foam.CLASS({
       name: 'displayWidth',
       value: foam.u2.layout.DisplayWidth.XL
     },
+    {
+      class: 'Boolean',
+      name: 'isDesktop',
+      expression: function(displayWidth) {
+        return displayWidth.minWidth > 320;
```
Comment (sarthak-marwaha) [inline src/foam/nanos/controller/ApplicationController.js]: Should be using the DisplayWidth enum, we should also do this in the view that needs this change, a generalized description of isDesktop is not going to work for many components

**Review by sarthak-marwaha — CHANGES_REQUESTED** [review]

File: src/foam/u2/view/LoginView.js [hunk 2]
```diff
@@ -217,7 +218,10 @@ foam.CLASS({
     {
       class: 'foam.u2.ViewSpec',
       name: 'leftView',
-      documentation: 'Allows using U2 views as left half of the login page, takes precedence over imgPath'
+      documentation: 'Allows using U2 views as left half of the login page, takes precedence over imgPath',
+      factory: function() {
+        return this.ctrl?.loginView?.leftView;
```
Comment (microArtur) [inline src/foam/u2/view/LoginView.js:223]: if this a proper way?

Comment (sarthak-marwaha) [reply src/foam/u2/view/LoginView.js:223]: Why is this needed?

File: src/foam/nanos/controller/ApplicationController.js [hunk 3]
```diff
@@ -224,6 +225,13 @@ foam.CLASS({
       name: 'displayWidth',
       value: foam.u2.layout.DisplayWidth.XL
     },
+    {
+      class: 'Boolean',
```
Comment (sarthak-marwaha) [inline src/foam/nanos/controller/ApplicationController.js]: We dont need this anymore

File: src/foam/nanos/theme/Theme.js [hunk 4]
```diff
@@ -240,6 +240,11 @@ foam.CLASS({
       section: 'images',
       writePermissionRequired: true
     },
+    {
+      class: 'Boolean',
+      name: 'shouldResizeLogo',
+      documentation: 'Enables switching between largeLogo and logo'
```
Comment (sarthak-marwaha) [inline src/foam/nanos/theme/Theme.js]:
```suggestion
      documentation: 'Enables switching between largeLogo and logo in ApplicationLogoView'
```

File: src/foam/u2/view/LoginView.js (hunk 2 above)
Comment (microArtur) [reply src/foam/u2/view/LoginView.js:223]: when I clicked "sign in" button in top left corner from the unauthenticated-transfer, the view on the left is LoginView and the menuid is `sign-in`

Comment (sarthak-marwaha) [reply src/foam/u2/view/LoginView.js:223]: Right, okay, this is fine

**Review by sarthak-marwaha — APPROVED** [review]

**Pull request merged**